.github/CONTRIBUTING.md

```diff
@@ -8,20 +8,55 @@ Afterwards, you can either contribute icons or code.
+
+When designing icons, it's important to strike a balance with the level of detail. While some icons can be highly detailed, it's acceptable to remove certain details as long as the icon remains recognizable and stays true to its original concept. In some cases, you may need to completely rethink the icon to achieve a minimalist design.
+
+A great example of this is the Subway Surfers icon as seen in the example above. In Lawnicons, it was reimagined by using the graffiti-styled "S" from the game's logo instead of depicting the character, which would have required intricate shapes and varying stroke thicknesses.
+
+![image](https://github.com/LawnchairLauncher/lawnicons/assets/49114212/5f0607ae-2daf-4b69-9d59-799f20d4caac)
+
+In contrary, it's also important to avoid too few details, as they may not be easily recognizable. Take the Headspace icon, for instance, which consists only of a circle. This may pose a problem for users since Lawnicons are not colored, and a simple circle on its own is not easily identifiable. Whenever possible, incorporate additional details that align with the app's design language. 
+
+In the Headspace example above, the circles usually have a face representing mood. By adding one of those faces to the circle, you can make the icon more recognizable.
+
+### Creating Icons 4/5 — Maintaining Consistency
+
+![image](https://github.com/LawnchairLauncher/lawnicons/assets/49114212/afb9c812-9dc4-4245-bbcd-4983852c5f40)
+
+All icons must adhere to an outline (not filled) to maintain a consistent theme. However, fill can be used sparingly for small details that require minimal contrast or for elements that cannot be accurately recreated with an outline. 
+
+The key is to ensure that any filled elements do not overwhelm the overall icon design and align with the outlined style to create a harmonious and coherent icon set.
+
+### Creating Icons 5/5 — Maintaining Balance
+
+![image](https://github.com/LawnchairLauncher/lawnicons/assets/49114212/e90b7936-195c-4fed-a7ef-ef2dcbcd2221)
+
+Sharp contrast occurs when there is a drastic change in stroke thickness without a smooth transition. For instance, using a *12px* stroke and suddenly decreasing it to *8px* creates an unbalanced visual effect in the final icon.
+
+To avoid sharp contrast, you can follow either of these two methods:
+
+1. **Uniform Stroke:** Instead of jumping between stroke thicknesses, use the next immediate increment in width. For example, transition from *12px* to *10px*.
+2. **Gradual Transition:** Rather than making a large jump, introduce an intermediate thickness. By going from *12px* to *10px* and then to *8px* you create a gradual transition or gradation. This approach helps maintain visual balance in your icon design.
+
+For additional information on designing icons and samples, see [the Lawnicons Figma repository](https://www.figma.com/community/file/1227718471680779613) made by [Grabstertv](https://github.com/Grabstertv).
```

Comment:
I'd suggest moving this text to the first heading (if you follow my first review comment). This paragraph doesn't really fit in the “Maintaining Balance” heading.